backend/Tournament/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.utils import timezone
+
+from User.models import User
+from .tournamentState import TournamentState
+
+# Create your models here.
+class Tournament(models.Model):
+	host_user = models.ForeignKey(User, related_name='tournament_hosts', on_delete=models.CASCADE)
+	name = models.CharField(max_length=30, null=True, blank=True, default=None)
```

Comment:
I see the 30 for char length a few times. Would be nice to have such magical values in a constants object or file, where they are easy to find and look up. Then I can catch them on the frontend already

Reply:
sure, but let's do it in some other pr